nbxplorer/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "nbxplorer.fullname" . }}-test-connection"
+  labels:
+    {{- include "nbxplorer.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
```

Comment:
Maybe make this test better. Is there some kind of readiness flag or acceptance criteria that we can check for?